Mozillians_Repositioning.md

```diff
@@ -0,0 +1,233 @@
+# Mozillians.org Repositioning Architecture
+
+* [Mozillians.org Services](#mozilliansorg-services)
+  + [Technologies (Common)](#technologies-common)
+  + [New Front-End](#new-front-end)
+  + [Mozillians.org](#mozilliansorg)
+  + [Search Service](#search-service)
+  + [Profile Publisher](#profile-publisher)
+  + [Orgchart Service](#orgchart-service)
+* [Integrity Checker](#integrity-checker)
+* [CIS requirements](#cis-requirements)
+  + [Profile Updates from Mozillians](#profile-updates-from-mozillians)
+  + [Profile Updates from external Sources](#profile-updates-from-external-sources)
+
+
+## Mozillians.org Services
+
+### Technologies (Common)
+
+#### GraphQL
+
+We stated a discussion around whether we want to test [GraphQL](https://graphql.org/) as our API
+query language. Could be a great fit and we plan to give it a shot.
+
+#### Node, Python, Rust
+
+We want to use whatever technology fits our needs. If it makes sense to write a small service in
+in Node or Rust we encourage it.
+
+
+### New Front-End
+
+A new front-end to render the new pages:
+
+- profile view
+- (edit profile section)
+  TBD if we do this _inline_ or as separate page
+- search result page
+- org chart
+
+#### Technologies
+
+- We plan to implement the new front-end using [Vue](https://vuejs.org).
+- We use components throughout across all pages
+- We plan to use global CSS instead of using components because we embrace the
+  [cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade)
+- Server side rendering is put on hold for now
+
+#### Questions
+
+Will we support IE11?
```

Comment:
i dont think its needed. does NLX support IE11? @hidde

Reply:
i actually just read one of the wontfix'd issues that says we dont support them well https://github.com/mozilla-iam/auth0-custom-lock/issues/124

Reply:
I'd love us to take a _progressive enhancement_ view on browser support, this is also what I try to do in NLX. 

It's ‘an escalator is still stairs when the power is off’ applied to web development, in our case it would that this works for anyone: search people, look at and edit profiles, figure out people's phone number, etc. In short: access the content. But it doesn't need to look/behave the same (so ok if in IE11, autocomplete doesn't work or is less smart; not ok if you can't even see results to your query or see a phone number in IE11).